test/unit/validation/test_issues_generation.py

```diff
@@ -5,13 +5,13 @@
 import requests
 from unittest.mock import patch
 
-from validation.schema import SchemaValidator
+from validation.json import JsonSchemaValidator
```

Comment:
If we change the name I would change it to `JSONValidator` as it is not validating the schema. With Isuru we realised that it is a bad name of that service and we are going to change it.